src/status_im2/contexts/emoji_picker/view.cljs

```diff
@@ -210,17 +201,16 @@
                                       :should-update-active-category? (nil? @filtered-data)}))]
     (fn [sheet-opts]
       [:f> f-view
-       (merge sheet-opts
-              {:render-emojis?            render-emojis?
-               :search-text               search-text
-               :on-change-text            on-change-text
-               :clear-states              clear-states
-               :filtered-data             @filtered-data
-               :set-scroll-ref            set-scroll-ref
-               :on-select                 on-select
-               :on-viewable-items-changed on-viewable-items-changed
-               :active-category           active-category
-               :scroll-ref                scroll-ref})])))
+       (assoc sheet-opts
```

Comment:
`assoc` instead of `merge` since we only want to add some keys to an existing map